manage

```diff
@@ -275,6 +275,17 @@ elif [ "$1" == 'install' ] || [ "$1" == 'uninstall' ];then
     mv "$logfile" "$(echo "$logfile" | sed 's+-incomplete-+-success-+g')" #rename logfile to indicate it was successful
     
   else #if app failed to install/uninstall
+
+    #remove dummy deb if app failed to install (to avoid dummy debs being left on a users install for broken apps)
+    if [[ ${mode} == "install" ]] && [[ "$(app_type "$app")" == standard ]]; then
+      local package_name="$(app_to_pkgname "$app")"
+      local output="$(sudo -E apt purge -y $package_name 2>&1 | less_apt | tee /dev/stderr)"
```

Comment:
Why run `apt purge` rather than `purge_packages`? Also this may run unnecessarily if the dummy deb is not installed, so we need to add a check for that.

Reply:
an unnecessary run isn't a big deal, but yeah that can be checked

I ran apt purge because that is what you ran in your script. It mimics the previous functionality by moving it to the manage script only when the app fails while not having the drawbacks of purging the dummy deb unnecessarily during a non-fatal install_packages run

```
  #If this app's dummy deb is already insalled, uninstall it
  if package_installed "$package_name" ;then
    status "Reinstalling the $package_name package..."
    apt_lock_wait
    local output="$(sudo -E apt purge -y $package_name 2>&1 | less_apt | tee /dev/stderr)"
    if [ $? != 0 ];then
      echo "$output"
      error "Failed to purge dummy deb ($package_name)"
    fi
  else
    status "Installing the $package_name package..."
  fi
  ```
  
  the main difference between running what I have written now and doing purge_packages is purge_packages will do an apt autoremove. 
I think an autoremove is fine for a failed first time install but I don't think its a good idea for a failed update.